src/app/admin/components/ProjectSubmissions.tsx

```diff
@@ -0,0 +1,103 @@
+'use client';
+
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import Link from 'next/dist/client/link';
+
+export interface Project {
+    id: string;
+    title: string | null;
+    subtitle: string | null;
+    description: string | null;
+    instructions: string | null;
+    requirements: string | null;
+    imageURL: string | null;
+    status: string | null;
+    deadline: Date | null;
+    startDateTime: Date | null;
+    endDateTime: Date | null;
+    createdAt: Date | null;
+    updatedAt: Date | null;
+    projectTimeline: string | null;
+    repoURL: string | null;
+}
+
+export interface AdminProfile {
+    userId: string;
+    displayName: string;
+    bio: string | null;
+    imageURL: string | null;
+    adminRole: 'Reg' | 'Mod' | 'Super' | 'idk' | null;
+    contactEmail: string | null;
+}
+
+export interface ProjectSubmission {
+    id: string;
+    projectId: string;
+    submittedOn: Date | null;
+    status: 'submitted' | 'under-review' | 'approved' | null;
+    reviewedOn: Date | null;
+    reviewedBy: string;
+    notes: string | null;
+
+    project: Project;
+    reviewer: AdminProfile;
+}
```

Comment:
you can use RouterOutputs instead of creating the interfaces here
```suggestion
type ProjectSubmissions = RouterOutputs['projectSubmission']['getAll'][number];
```